modules/govuk_jenkins/templates/jobs/deploy_terraform_govuk_aws.yaml.erb

```diff
@@ -0,0 +1,72 @@
+---
+- scm:
+    name: govuk-aws_Deploy_Terraform_Govuk_AWS
+    scm:
+        - git:
+            url: git@github.com:alphagov/govuk-aws.git
+            branches:
+                - master
+- job:
+    name: Deploy_Terraform_Project
+    display-name: Deploy_Terraform_Project
+    project-type: freestyle
+    description: |
+        Deploy GOV.UK AWS Infrastructure using Terraform.
+
+        For available projects to deploy, please see the <a href=https://github.com/alphagov/govuk-aws/tree/master/terraform/projects>Terraform repository</a>
+
+    properties:
+        - github:
+            url: https://github.com/alphagov/govuk-aws/
+    scm:
+        - govuk-aws_Deploy_Terraform_Govuk_AWS
+    wrappers:
+        - ansicolor:
+            colormap: xterm
+        - build-name:
+          name: '${ENV,var=ENVIRONMENT} ${ENV,var="STACKNAME"} ${ENV,var="PROJECT"} ${ENV,var="COMMAND"}'
+    builders:
+        - shell: |
+            ./jenkins.sh
+
+    parameters:
+        - string:
+            name: AWS_ACCESS_KEY_ID
+            description: Your AWS access key ID
+            default: false
+        - password:
+            name: AWS_SECRET_ACCESS_KEY
+            description: Your AWS secret access key
+            default: false
+        - password:
+            name: AWS_SESSION_TOKEN
+            description: The AWS session token
+            default: false
+        - choice:
+            name: COMMAND
+            description: Choose whether to plan (default), apply or destroy
```

Comment:
Where do we set `plan` as default? Or is that a Terraform thing?

Reply:
It's just how the Jenkins dropdown works (the top is always the default)